site/en/tutorials/generative/autoencoder.ipynb

```diff
@@ -159,27 +159,29 @@
       },
       "outputs": [],
       "source": [
-        "latent_dim = 64 \n",
-        "\n",
-        "class Autoencoder(Model):\n",
-        "  def __init__(self, latent_dim):\n",
+        "class Autoencoder(keras.Model):\n",
```

Comment:
`keras.Model` isn't imported. `Model` is already imported (`from tensorflow.keras.models import Model`), so you could leave this as it was.